### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by ssatagopan**

### CODE REVIEW

**Review by kilbergr — APPROVED** [review]
LGTM 🚢

File: .github/ISSUE_TEMPLATE/change-request.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+Name of application that is impacted by this change
+
+## Purpose
+What is the problem or need you are trying to solve with this request or 
+What is the outcome you would like once this change is completed?
+
+## Description of this Change
+Detailed information about the change being requested.
+
+_Is this change a configuration change or a software change?_
```
Comment (jduss4) [inline .github/ISSUE_TEMPLATE/change-request.md:23]: These will appear on the same line, at the moment:

image

Maybe consider making them bullet points?

File: .github/ISSUE_TEMPLATE/change-request.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+_Does this change require data schema migrations?_
+_Does this change fix a security vulnerability?_
+_What is the expected outcome and behavior?_
+
+## User Impact
+Who does this impact? (i.e. Developers, application administrators, application end users)
+
+## Timeline
+Is there a deadline for this change?
```
Comment (jduss4) [inline .github/ISSUE_TEMPLATE/change-request.md:32]: Would it be helpful to try to discover any other time concerns beyond actual deadlines?  Maybe "deadline or time considerations" or something

**Review by jduss4 — APPROVED** [review]
I can't think of any other questions you'd want to ask, looks great!  I added just a couple small comments which you can take or leave :)

**Review by rahearn — APPROVED** [review]
Verified that this contains the contact information of the requester as we discussed in the QASP implementation review last week. Otherwise deferring to the reviews already completed.

**Pull request merged**